README.md

```diff
@@ -1,6 +1,10 @@
 # amazon-vpc-cni-k8s
 Networking plugin for pod networking in [Kubernetes](https://kubernetes.io/) using [Elastic Network Interfaces](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html) on AWS.
 
+## note
+
+* STATUS: Preview. Do not use for production workloads while in preview 
```

Comment:
Make Status the bold heading.  Let's use more canonical status terms than "Preview".

```
## Status

 **Alpha**  This is an experimental release as part of the [Amazon EKS](https://aws.amazon.com/eks/) Preview.
Interfaces and functionality may change. Expect bugs (and please help us squash them).
DO NOT use for production workloads.
```